autoload/ddu/ui/ff/filter.vim

```diff
@@ -127,7 +127,12 @@ function! s:init_buffer(name, params) abort
         \ a:params.split ==# 'floating'
         \ || a:params.filterSplitDirection ==# 'floating'
 
-  const bufnr = ('ddu-ff-filter-' .. a:name)->bufadd()
+  const bufname = 'ddu-ff-filter-' .. a:name
+  if bufname->bufexists()
+    return
+  endif
+
+  const bufnr = bufname->bufadd()
```

Comment:
When `ddu-ff-filter-a:name` buffer exists, prevent to create more than two the buffers.